Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -74,7 +74,19 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     break;
   case COL_TITLE:
     if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
-      return QString::fromStdString(game.GetName());
+    {
+      QString name = QString::fromStdString(game.GetName());
+      const int disc_nr = game.GetDiscNumber() + 1;
+      if (game.GetDiscNumber() != 0)
+      {
+        if (!name.contains(QRegExp(QStringLiteral("disc ?%1").arg(disc_nr), Qt::CaseInsensitive)))
+        {
+          name.append(game.GetDiscNumber() != 0 ? tr(" (Disc %1)").arg(disc_nr) :
```

Comment:
You've already checked `game.GetDiscNumber() != 0` four lines above, so you don't need a ternary operator here.